### DISCUSSION TITLE
kong start -c /etc/kong/kong.conf  error!

### DISCUSSION THREAD

**Discussion by Canon4G** [Kong/kong#9235] [Help] [answered]
MacOs install kong by brew

/etc/kong/konf.conf content:
database = postgres             # Determines which of PostgreSQL or Cassandra

pg_host = 127.0.0.1             # Host of the Postgres server.
pg_port = 5432                    # Port of the Postgres server.
pg_timeout = 5000               # Defines the timeout (in ms), for connecting,

pg_user = kong                    # Postgres user.
pg_password = kong             # Postgres user's password.
pg_database = kong             # The database name to connect to.

pg_ssl = off                         # Toggles client-server TLS connections
                                
pg_ssl_verify = off               # Toggles server certificate verification if

my bash_profile:
  export PATH=/usr/local/openresty/nginx/sbin:$PATH
  export PATH=/usr/local/openresty/bin:$PATH

  export PATH="/usr/local/Cellar/kong/2.8.1/bin:$PATH"

but when i use command 'kong start -c /etc/kong/kong.conf'
the terminal return msg：
2022/08/12 14:16:00 [warn] ulimit is currently set to "256". For better performance set it to at least "4096" using "ulimit -n"
Error: nginx configuration is invalid (exit code 1):
nginx: [emerg] unknown directive "lua_kong_load_var_index" in /usr/local/opt/kong/nginx-kong.conf:50
nginx: configuration file /usr/local/opt/kong/nginx.conf test failed

  Run with --v (verbose) or --vv (debug) for more details

i have this problem

**Comment by chronolaw** [comment, marked as answer]
The directive `lua_kong_load_var_index` is implemented by nginx C module `lua-kong-nginx-module`,
and it is integrated with Kong.

You can try `nginx -V` to check if this module is enabled.

**Reply by Canon4G** [reply]
i user this command 'kong start -c /etc/kong/kong.conf --v'

return me this msg:
2022/08/12 15:18:46 [debug] found OpenResty 'nginx' executable at /usr/local/openresty/nginx/sbin/nginx
2022/08/12 15:18:46 [debug] testing nginx configuration: KONG_NGINX_CONF_CHECK=true /usr/local/openresty/nginx/sbin/nginx -t -p /usr/local/opt/kong -c nginx.conf

why not load /etc/kong/kong.conf

**Reply by kikito** [reply]
`/etc/` is protected in MacOS. If you install things in homebrew then you must use the homebrew prefix, which is in `/usr/local/opt/<your-app>`.

**Comment by GavinXu520** [comment]
So was this problem finally solved?
